src/ml_tooling/transformers/__init__.py

```diff
@@ -17,4 +18,5 @@
            'Binner',
            'Renamer',
            'DateEncoder',
-           'FreqFeature']
\ No newline at end of file
+           'FreqFeature',
+           'DFStandardScaler']
```

Comment:
```suggestion
           'DFStandardScaler',
           ]
```
Jeg havde selv glemt det, men for nemmere git diffing er det en god ide at have en trailing comma på listen, så git ser det kun som en linjeændring hvis man tilføjer noget nyt istedenfor to linjer.